ambry-router/src/test/java/com.github.ambry.router/DeleteManagerTest.java

```diff
@@ -148,6 +149,11 @@ public void onCompletion(Void result, Exception exception) {
             for (Future future : futures) {
               future.get(AWAIT_TIMEOUT_SECONDS, TimeUnit.SECONDS);
             }
+            long waitStart = SystemTime.getInstance().milliseconds();
```

Comment:
why is this needed here? does the `DeleteManagerTest` test any composite blob deletes?